src/ui.data-table/storybook/stories.js

```diff
@@ -203,6 +203,35 @@ const EmptyTemplate = (args) => ({
 export const Default = DefaultTemplate.bind({});
 Default.args = {};
 
+export const RowClasses = DefaultTemplate.bind({});
+RowClasses.args = {
+  columns: [
+    { key: "id", label: "ID" },
+    { key: "name", label: "Name" },
+    { key: "status", label: "Status" },
+    { key: "amount", label: "Amount" },
+  ],
+  row: (index) => ({
+    id: index + 1,
+    name: `Item ${index + 1}`,
+    status: index % 3 === 0 ? "active" : index % 3 === 1 ? "pending" : "inactive",
+    amount: Math.floor(Math.random() * 1000),
```

Comment:
the class should be applied in every td we want like. adding the separate props is not a right solution.

`amount: {value: Math.floor(Math.random() * 1000), class: "bg-green-100"}`.

In your solution there is no possible to apply classes to the one separated td of the row.